### PR TITLE
multiple pages: add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by Proscream** [comment]
I think I made a mistake. The old branches were reused.

**Comment by schneiderl** [comment]
I don't know anything about Korean but shouldn't we have the examples translated too?

**Comment by Proscream** [comment]
@schneiderl do you mean to translate "echo {{His network looks good}} | alex --stdin" in 'alex.md' ?

**Comment by schneiderl** [comment]
@Proscream I've just realized what a dumb thing I've said. 

I was talking about translating the examples on `7z.md`, for instance. 
But there is no way on translating that since 7z command wouldn't recognize Korean wording. 
lol
Sorry about that and thank you for the quick response 😄

**Comment by Proscream** [comment]
@schneiderl If there is a conflict here because of duplicate pages, can I leave a PR again?

**Comment by Proscream** [comment]
@mebeim If trying to translate {{user_token}, perhaps {{유저_토큰}} It will only be. The word is a English pronunciation of "user_token" in Korean.

**Comment by mebeim** [comment]
@Proscream sorry I wasn't talking specifically about `{{user_token}}`, I was talking about the tokens in the pages in general. I see you kept all of them in English. If that is intended then that's fine. Otherwise the ones that makes sense to translate should be translated.

**Comment by Proscream** [comment]
@mebeim thank you for a comment. I knew all the translations I could do, but its not. Possible strings have been translated.

File: pages.ko/common/ag.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# ag
+
+> The Silver Searcher.
+> ack과 비슷하지만, 더 빠르다. 
+> 더 많은 정보: <https://github.com/ggreer/the_silver_searcher>.
+
+- Find files containing "foo"를 포함하고 있는 파일들을 찾고, 내용에서 일치하는 행을 출력:
```
Comment (einverne) [inline pages.ko/common/ag.md]: Find files containing "foo" 

Is this phrase needed here?

Comment (Proscream) [reply pages.ko/common/ag.md]: Oh, its my mistake. I'll modify that.

**Pull request merged**